doc/site/zh-cn/guides/thread-primer/ipv6-addressing.md

```diff
@@ -0,0 +1,231 @@
+# IPv6 寻址
+
+让我们来看一下 Thread 如何识别网络中的每个设备，以及设备间用何种类型的地址进行相互通信。
+
+Key Term: 在本入门教程中，术语“接口（interface）”用于标识网络内 Thread 设备的端点。通常，单个 Thread 设备具有单个 Thread 接口。
+
+## 域
+
+<figure class="attempt-right">
+<a href="../images/ot-primer-scopes_2x.png"><img src="../images/ot-primer-scopes.png" srcset="../images/ot-primer-scopes.png 1x, ../images/ot-primer-scopes_2x.png 2x" border="0" alt="OT Scopes" /></a>
+</figure>
+
+Thread 网络中有三种域用于单播寻址：
+
+* Link-Local — 所有接口通过单次无线电传输都是可到达的
+* Mesh-Local — 所有接口在同一 Thread 网络中都是可到达的
+* Global — 所有接口在 Thread 网络外都是可到达的
+
+前两个域与 Thread 网络指定的 Prefix（前缀）相对应。Link-Local 的 Prefix 为 `fe80::/16`，Mesh-Local 的 Prefix 为 `fd00::/8`。
+
+<h2 style="clear:right">单播</h2>
+
+单个 Thread 设备可以通过多种 IPv6 单播地址来进行标识。每种地址都有不同的功能（基于域和用例）。
+
+在介绍每种类型之前，让我们先了解一个共同的概念，它叫作 RLOC（Routing Locator，路由定位器）。RLOC 根据 Thread 接口在网络拓扑中的位置来对其进行标识。
+
+### 如何生成 RLOC
+
+所有设备都获得一个 Router ID 和一个 Child ID。每个 Router 维护一个包含其所有子节点的表，两个 ID 的组合唯一地标识拓扑中的设备。例如，请考虑以下拓扑中高亮的节点，其中 Router（五边形）上的数字是 Router ID，End Device（圆形）上的数字是 Child ID：
```

Comment:
@jwhui @Vyrastas , I think the text `where the number on a Router (pentagon) is the Router ID, and the number on an End Device (circle) is the Child ID` doesn't match the picture which demonstrates the RLOC16,   but not Router ID or Child ID.

Reply:
@librasungirl Can you be more specific on what doesn't match?  I'm not following.

Reply:
I think the original text can be removed `where the number on a Router (pentagon) is the Router ID, and the number on an End Device (circle) is the Child ID`.
Because this sentence does not clearly express what it should mean. In addition, the relationship between RLOC16 and Router ID and Child ID has been correctly described later.
@librasungirl @Vyrastas

Reply:
@Vyrastas 
 See https://openthread.io/guides/thread-primer/ipv6-addressing#how_a_routing_locator_is_generated
To be specific, in the picture, the number on a Router (pentagon) is 0x400, it is RLOC 16, not Router ID; The number on an End Device (circle) is 0x0401, it is RLOC16, not Child ID.

Guessing the original intention here is to highlight the RLOC16, not Router ID or Child ID? as the header for this part is `How a Routing Locator is generated`

Reply:
@librasungirl #5163

Reply:
@librasungirl , I think the original text is technically correct. The number _**on**_ a Router (pentagon) is the Router ID however, the number _**to the side of**_ a Router (pentagon) is the RLOC16. Similarly for the Child ID. How the Router ID and Child ID map to RLOC16 is describe in the text below the image. Thoughts?

Reply:
Oh, yes, you are right. Somehow the RLOC16 seems catching more eyes when reading.

Reply:
The meaning of the original text should be the numbers in the graphics rather than the numbers on the sides. So is it more appropriate to use `in/inside` than `on`? @jwhui @librasungirl 

where the number **in/inside** the Router (pentagon) is the Router ID, and the number **in/inside** the End Device (circle) is the Child ID

Reply:
Not sure if the confusion is just out of some reading habits, the original text seems fine to me after @jwhui 's clarification.
I am also fine with the proposal by @dsyx to avoid some confusion for readers, like me, at first glance; or put it in another way
> where the number in the pentagon (a Router) is the Router ID, and the number in the circle (an End Device) is the Child ID.

No strong preference from me, your thoughts? @Vyrastas

Reply:
Sorry, I missed this conversation.  What @dsyx did in #5163 is fine (replace "on" with "in"), if that helps clear up confusion for some readers.